x/pool-incentives/keeper/grpc_query.go

```diff
@@ -136,7 +170,7 @@ func (q Querier) ExternalIncentiveGauges(ctx context.Context, req *types.QueryEx
 
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	store := sdkCtx.KVStore(q.Keeper.storeKey)
-	prefixStore := prefix.NewStore(store, []byte("pool-incentives"))
+	prefixStore := prefix.NewStore(store, []byte("pool-incentives/"))
```

Comment:
does this mean `prefixStore` wasn't returning correct data before? and  adding "/" solved?

Reply:
This wasn't broken. I added it to prevent reading from another index with the same prefix:
https://github.com/osmosis-labs/osmosis/blob/95f89de0c26440e7ea041ed59e3be698fff2d8d2/x/pool-incentives/types/key.go#L30